mod-orders-storage/schemas/suffix.json

```diff
@@ -2,9 +2,6 @@
   "$schema": "http://json-schema.org/draft-04/schema#",
   "description": "Purchase order number suffix",
   "type": "object",
-  "extends" : {
-    "$ref" : "../../common/schemas/entity.json"
-  },
```

Comment:
If entity.json is not using in other places, I think we can delete it as well

Reply:
Thanks, removed